cluster/gce/gci/health-monitor.sh

```diff
@@ -71,10 +71,10 @@ function kubelet_monitoring {
   sleep 120
   local -r max_seconds=10
   local output=""
-  while [ 1 ]; do
+  while true; do
     if ! output=$(curl -m "${max_seconds}" -f -s -S http://127.0.0.1:10248/healthz 2>&1); then
       # Print the response and/or errors.
-      echo $output
+      echo "${output}"
```

Comment:
avoid re-splitting etc by quoting the string. not a big deal here but technically a correctness fix.